### DISCUSSION TITLE
Is there any way to only install dependencies a single project in a workspace?

### DISCUSSION THREAD

**Discussion by alvis** [pnpm/pnpm#6744] [Q&A] [answered]
I have a service maintained in a workspace among 2x other projects. Currently when I deploy that service, the `pnpm i` step would install all dependencies of all projects, not only the service of interest. Is there anyway to reduce the build time and bandwidth cost by ignoring other unnecessary dependencies from other projects?

**Comment by gluxon** [comment, marked as answer]
Hey @alvis, this was one of the biggest reasons my company chose to use pnpm. It's possible to perform filtering on the `pnpm install` command: https://pnpm.io/filtering

```
pnpm install --filter @example/workspace-project-a
```

**Reply by alvis** [reply]
@gluxon Thank you so much. I completely overlooked it. :)

**Reply by gluxon** [reply]
Glad this helped!
